db/migrate/20160526142456_add_payment_mode_to_tickets.rb

```diff
@@ -0,0 +1,5 @@
+class AddPaymentModeToTickets < ActiveRecord::Migration
+  def change
+    add_column :tickets, :payment_mode, :string
```

Comment:
By adding the column like this, all old tickets will have a NULL `payment_method`. Is that valid? Should we do something about that?